indra/newview/llpanelgroupbulkban.cpp

```diff
@@ -163,27 +161,26 @@ void LLPanelGroupBulkBan::submit()
 	// remove already banned users and yourself from request.
 	std::vector<LLAvatarName> banned_avatar_names;
 	std::vector<LLAvatarName> out_of_limit_names;
-	bool banning_self = FALSE;
-	std::vector<LLUUID>::iterator conflict = std::find(banned_agent_list.begin(), banned_agent_list.end(), gAgent.getID());
+	bool banning_self{ false };
+	std::vector<LLUUID>::iterator conflict = std::find(banned_agent_list.begin(), banned_agent_list.end(), gAgentID);
```

Comment:
Interesting - is there ever a case in which `gAgentID` does not exactly track `gAgent`?

Reply:
No. I just replaced it because I stumbled over it and it saves a function call.